Http/Controllers/Api/MediaController.php

```diff
@@ -37,74 +45,102 @@ public function __construct(FileService $fileService, FileRepository $file, Imag
     public function all()
     {
         $files = $this->file->all();
-
+        
         return [
             'count' => $files->count(),
-            'data' => $files,
+            'data' => $files
         ];
     }
 
     /**
      * Store a newly created resource in storage.
-     * @param  UploadMediaRequest $request
+     * 
+     * @param UploadMediaRequest $request            
      * @return Response
      */
     public function store(UploadMediaRequest $request)
     {
         $savedFile = $this->fileService->store($request->file('file'));
-
+        
         if (is_string($savedFile)) {
-            return Response::json(['error' => $savedFile], 409);
+            return Response::json([
+                'error' => $savedFile
+            ], 409);
         }
-
+        
         event(new FileWasUploaded($savedFile));
-
+        
         return Response::json($savedFile->toArray());
     }
 
     /**
      * Link the given entity with a media file
-     * @param Request $request
+     * 
+     * @param Request $request            
      */
     public function linkMedia(Request $request)
     {
         $mediaId = $request->get('mediaId');
         $entityClass = $request->get('entityClass');
         $entityId = $request->get('entityId');
         $order = $request->get('order');
-
+        
         $entity = $entityClass::find($entityId);
         $zone = $request->get('zone');
-        $entity->files()->attach($mediaId, ['imageable_type' => $entityClass, 'zone' => $zone, 'order' => $order]);
-        $imageable = DB::table('media__imageables')->whereFileId($mediaId)->whereZone($zone)->whereImageableType($entityClass)->first();
+        $entity->files()->attach($mediaId, [
+            'imageable_type' => $entityClass,
+            'zone' => $zone, 
+            'order' => $order
+        ]);
+        $imageable = DB::table('media__imageables')->whereFileId($mediaId)
+            ->whereZone($zone)
+            ->whereImageableType($entityClass)
+            ->first();
         $file = $this->file->find($imageable->file_id);
-
-        $thumbnailPath = $this->imagy->getThumbnail($file->path, 'mediumThumb');
-
+        
+        if ($file->mimetype == 'video/mp4' || $file->mimetype == 'video/ogv' || $file->mimetype == 'video/webm') {
```

Comment:
Just a small nitpick (you do not need to make this change if you don't want to), I would use something like `strpos($file->mimetype, 'video') !== 0` so you catch all possible `video/*` mimetypes. It also saves you a bit of typing. :)